components/ui/Input.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from "react";
+
+type InputProps = {} & React.InputHTMLAttributes<HTMLInputElement>;
+
+const defaultClassName =
+  "border-1 rounded-md border-transparent bg-anti-flash-white outline-none px-3 items-center";
+
+const Input = React.forwardRef<HTMLInputElement | null, InputProps>(
+  (props, ref) => {
+    const inputRef = React.useRef<HTMLInputElement | null>(null);
+    const [focused, setFocused] = React.useState(false);
+
+    let { className, ...restProps } = props;
+    className = `${defaultClassName} ${className ?? ""}`;
+    const isNumber = props.type === "number";
+
+    if (isNumber) {
+      className += " font-mono";
```

Comment:
Don't think we should be using the mono font anymore, @mk26071980 should be able to confirm